packages/styles/scss/components/structured-list/_structured-list.scss

```
@@ -200,6 +199,27 @@
     fill: $icon-primary;
   }
 
+  @if not(
+    enabled('enable-v12-structured-list-visible-icons') or
+      $enable-v12-structured-list-visible-icons
```

Both of these need to be false I think
```suggestion
  @if not(
    enabled('enable-v12-structured-list-visible-icons') and
      $enable-v12-structured-list-visible-icons
```